### PR TITLE
fix: support wallet=main param based on network

### PR DESCRIPTION
**Pull request by arirubinstein**
If network != main, use the main branch of the wallet in the redirect

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ const networkConfigUrl = (() => {
   }
 })();
 
+const walletParam = (networkConfigUrl !== "main" ? "wallet=main" : "");
```
Comment (samsiegart) [inline src/main.ts]: The url is the full url, you want `agoricNetName` above, you can just pull that outside the scope of the function so you can reference it here

File: src/main.ts [hunk 2]
```diff
@@ -47,7 +48,7 @@ const tryRedirect = async () => {
     setMessage('Not found: No Endorsement');
   } else {
     try {
-      const href = `https://${endorsedUI.value}.ipfs.cf-ipfs.com`;
+      const href = `https://${endorsedUI.value}.ipfs.cf-ipfs.com/?${walletParam}`;
```
Comment (samsiegart) [inline src/main.ts]: Nit: Move the "?" into the template literal so it doesn't show up unless "wallet=main" does

**Comment by arirubinstein** [comment]
thx, no testing ftw

File: src/main.ts [hunk 3]
```diff
@@ -9,17 +9,17 @@ const fetchRPCAddr = async (netconfigURL: string) => {
   return rpcAddrs[Math.floor(Math.random() * rpcAddrs.length)];
 };
 
-const networkConfigUrl = (() => {
-  const agoricNetName =
-    new URLSearchParams(window.location.search).get('network') ?? 'main';
+const agoricNetName = new URLSearchParams(window.location.search).get('network') ?? 'main';
```
Comment (samsiegart) [inline src/main.ts:12]: Note: This line is over 80 chars, but we don't have prettier setup in the repo so I should just do that. I've just been using the default vscode prettier.

**Review by samsiegart — APPROVED** [review]

File: src/main.ts [hunk 4]
```diff
@@ -9,17 +9,17 @@ const fetchRPCAddr = async (netconfigURL: string) => {
...
 
+const networkConfigUrl = (() => {
   if (agoricNetName === 'local') {
     return 'https://wallet.agoric.app/wallet/network-config';
   } else {
     return `https://${agoricNetName}.agoric.net/network-config`;
   }
 })();
 
+const walletParam = (agoricNetName !== "main" ? "?wallet=main" : "");
```
Comment (turadg) [inline src/main.ts]:
```suggestion
const walletParam = (agoricNetName == "main" ?  '' : '?wallet=main');
```

**Pull request merged**